powerquery-docs/query-folding-basics.md

```diff
@@ -1,115 +1,124 @@
 
-1. Only get the data from September 2012 by filtering the rows on the OrderDate column.
+- **Source**: Makes the connection to the data source. In this case it's a connection to a SQL Server database.
+- **Navigation**: Navigates to a specific table within the database.
+- **Removed other columns**: Selects which columns from the table to keep.
+- **Sorted rows**: Sorts the table using one or multiple columns using a descending order.
+- **Kept top rows**: Filters the table to only keep a certain number of rows from the top of the table.
 
-   ![Filtering the FactInternetSales table by the OrderDate column for only dates in the month of September 2012.](images/me-query-folding-basics-filter-values.png)
+The Power Query editor helps you add these steps, through a diverse set of interactions on its user interface, to create the M Script that produces the query output you want.
 
-   >[!NOTE]
-   > To learn more about how to filter rows by their values, go to [Filter values](filter-values.md).
+This set of step names is a friendly way to interpret the M script that Power Query has created for you. To view this full M script you can either go to the advanced editor window in the View tab or change the view of the formula bar to show the query script by going into the View tab and from the Layout group select the **Script view** to be *Query script*.
 
-2. Group by the OrderDate column and create a new aggregated column using the **Sum** operation on the SalesAmount column. Name this new column Total Sales Amount.
+![image that displays the full script and the corresponding names of the steps](media/query-folding-basics/m-script-applied-steps.png) 
 
-   ![Group by using the OrderDate column and aggregating by the SalesAmount column.](images/me-query-folding-basics-group-by.png)
+Most of the names found in the applied steps pane are also being used in the M script and in some cases they are wrapped around other characters. The Power Query editor always tries to show you a friendly name instead of the exact name being used inside your M script to help you interpret your query much easier.
 
-   >[!NOTE]
-    > To learn more about how to use the group by feature, go to [Grouping or summarizing rows](group-by.md).
+>[!NOTE]
+>Steps of a query are named using something called *identifiers* in the M language. Sometimes extra characters are wrapped around step names in M, but these aren’t shown in the applied steps. An example is the  *#"Kept top rows"* which is categorized as a *quoted identifier* because of these extra characters. A quoted identifier can be used used to allow any sequence of zero or more Unicode characters to be used as an identifier, including keywords, whitespace, comments, operators and punctuators. You can learn more about *identifiers* in the M language from the documentation on [lexical structure](https://docs.microsoft.com/powerquery-m/m-spec-lexical-structure#identifiers).
```

Comment:
2 times the word "used" in "A quoted identifier can be used used..."

Reply:
thanks @NickyvVr ! :) Ehren and the rest of the team are handling this article internally. Hope that you find this early draft helpful! the full-blown article should be available in the upcoming days if possible.